modules/rosapps/applications/devutils/vgafontedit/mainwnd.c

```diff
@@ -221,6 +222,34 @@ DoFileOpen(IN PMAIN_WND_INFO Info)
     }
 }
 
+static VOID
+MainWndOpenFile(IN PMAIN_WND_INFO Info, LPCWSTR File)
+{
+    PFONT_OPEN_INFO OpenInfo;
+
+    OpenInfo = HeapAlloc(hProcessHeap, HEAP_ZERO_MEMORY, sizeof(FONT_OPEN_INFO));
+    OpenInfo->pszFileName = HeapAlloc(hProcessHeap, 0, MAX_PATH);
+    lstrcpynW(OpenInfo->pszFileName, File, MAX_PATH);
```

Comment:
StringCb/CchCopy would be more appropriate. Also since a truncated file name doesn't make much sense, erroring in case of overflow would be good.

Reply:
OK.
See [fb8fe4a](https://github.com/reactos/reactos/commit/fb8fe4a08c4cfba02b019fab0667720c43173d1c).